frontend/src/features/whats-new/utils/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { isAfter } from 'date-fns'
+
+import { UserDto } from '~shared/types/user'
+
+import { FeatureUpdateList } from '../FeatureUpdateList'
+
+export interface shouldShowLatestFeatureUpdateNotificationProps {
```

Comment:
nit: name of function sounds like a variable instead of a function.
suggest `getShowLatestFeatureUpdateNotificationState`?